release/release_test.go

```diff
@@ -66,13 +65,7 @@ BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
 	return mockOSRelease
 }
 
-// Kernel magic numbers
-const (
-	wslfs int64 = 0x53464846
-	ext4  int64 = 0xef53
-)
-
-func mockWSLsetup(c *C, existsWSLinterop bool, existsRunWSL bool, filesystemID int64) func() {
+func mockWSLsetup(c *C, existsWSLinterop bool, existsRunWSL bool, filesystemName string) func() {
```

Comment:
This is pre-existing code so this is fine but the booleans args in this API are not great. When reading (down below where this is used): `defer mockWSLsetup(c, true, false, "btrfs")()` I have no idea what true, false mean so I need to go and look at the function itself first. Again, fine to keep as it already exists.

Reply:
I changed is so it looks like 
```go
defer mockWSLsetup(c, mockWsl{ExistsInterop: false, ExistsRunWSL: false, FsType: "ext4"})()
```

Is it ok or would you prefer I don't inroduce a new type?